frontend/src/mocks/handlers/link.ts

```diff
@@ -0,0 +1,20 @@
+import { rest } from 'msw';
+import type { TeamLink } from '~/types/link';
+
+const teamLink = [] as TeamLink[];
```

Comment:
복수의 팀 링크를 저장한다면, 변수를 복수형으로 하면 어떨까?
```suggestion
const teamLinks = [] as TeamLink[];
```

Reply:
좋아